config/database.php

```diff
@@ -13,7 +13,7 @@
     |
     */
 
-    'default' => env('DB_CONNECTION', 'mysql'),
+    'default' => env('DB_CONNECTION', 'sqlite'),
```

Comment:
rather than changing the config, you should just change `DB_CONNECTION=sqlite` in the `.env` file.